test/utils.tests.js

```diff
@@ -0,0 +1,45 @@
+const { File, processFileController } = require('../utils.js');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+const expect = chai.expect;
+chai.use(chaiAsPromised);
+
+const testFilePath = `${__dirname}/testFile.json`;
+const invalidFilePath = './invalidPath';
+
+describe('File Class', () => {
+  const newFile = new File(testFilePath);
+
+  it('it should be an object that containts a filename property', () => {
+    expect(newFile).to.be.an('object');
+    expect(newFile.filename).to.be.equal(testFilePath);
+  });
+
+  it('it should read a file and parse the json data and store in the filecontents property', async () => {
+    const results = await newFile.readFile();
+    expect(results)
+      .to.be.an('object')
+      .to.have.property('test');
+    expect(results.test).to.equal('testMessage');
```

Comment:
This is a good test, you're testing one of the expected behavior of this function.